relation_engine_server/test/test_pull_spec.py

```diff
@@ -0,0 +1,37 @@
+import unittest
+from unittest import mock
+import re
+
+from relation_engine_server.utils.pull_spec import download_specs
+from relation_engine_server.utils.wait_for import wait_for_api
+from spec.test.test_ensure_specs import ensure_borked_indexes
+
+
+class Test(unittest.TestCase):
```

Comment:
do you want to give the test class a more exciting name?

Reply:
Those are harder to remember when targeting specific test methods from the CLI ...

Reply:
`class Test_Pull_Spec` ain't so bad, is it?